### PR TITLE
[Bug Fix] Create Wrapper Function To Handle GUID Table Properties

### PR DESCRIPTION
**Pull request by nharper285**
## Summary of the Pull Request

_What is this about?_
It was found that the `onefuzz debug [node | task | etc.]` command was not properly working - users were not able to rdp into task vmss instances. 

The issue was due to how we were querying certain ORM tables. To resolve the issue, Ive created a wrapper functions that converts all `guid` argument types to `string`.

## PR Checklist
* [ ] Applies to work item: #2897
* [ ] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/onefuzz) and sign the CLI.
* [ ] Tests added/passed
* [ ] Requires documentation to be updated
* [ ] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #xxx

## Info on Pull Request

_What does this include?_

## Validation Steps Performed

_How does someone test & validate?_

### CODE REVIEW

**Review by chkeita — APPROVED** [review]

File: src/ApiService/ApiService/onefuzzlib/orm/Queries.cs [hunk 1]
```diff
@@ -8,11 +9,22 @@ public static class Query {
         // and handles escaping the interpolated values properly. It also handles quoting the values
         // where needed, so use {string} and not '{string}'.
 
+        public static string CreateQueryFilter(FormattableString input) {
```
Comment (mgreisen) [inline src/ApiService/ApiService/onefuzzlib/orm/Queries.cs:12]: Can we create some unit tests?

Comment (nharper285) [reply src/ApiService/ApiService/onefuzzlib/orm/Queries.cs:12]: Added unit test!

**Comment by Porges** [comment]
Looking good 👍

**Review by Porges — APPROVED** [review]

File: src/ApiService/ApiService/onefuzzlib/NodeOperations.cs [hunk 2]
```diff
@@ -239,7 +238,7 @@ sealed record NodeInfo(Node Node, Scaleset Scaleset, string InstanceId);
     public async Async.Task ReimageLongLivedNodes(Guid scaleSetId) {
         var timeFilter = Query.OlderThan("initialized_at", DateTimeOffset.UtcNow - INodeOperations.NODE_REIMAGE_TIME);
         //force ToString(), since all GUIDs are strings in the table
```
Comment (Porges) [inline src/ApiService/ApiService/onefuzzlib/NodeOperations.cs:240]: can remove this comment now

Comment (nharper285) [reply src/ApiService/ApiService/onefuzzlib/NodeOperations.cs]: Done.

**Review by mgreisen — APPROVED** [review]
:shipit:

**Review by tevoinea — APPROVED** [review]

**Pull request merged**